### PR TITLE
Marketplace: Discover page title and margin fixes

### LINKED ISSUE #65574: Improving Titles and Labels in Plugins Page
Besides obvious design improvements to the https://wordpress.com/plugins/ page (for which we can open a different issue), it's worth adding slightly better labelling and titles for more context, storytelling, and precision to categories, actions, and the whole marketplace/plugins page. These below are only suggestions @michaelpick, hopefully the intent comes across. 

- [ ] `Plugins you need to get your projects done` to something like `All the plugins to power and flex your site`

- [ ] `Top` is redundant, we use it everywhere so it becomes irrelevant. Add more storytelling, elevate each category. 
Not `Top paid plugins` but `Featured Paid Plugins`.
Not `Editor’s pick`, but `Our Developers Favorites` 
Not `Top free plugins`, but `The Free Essentials`
Other examples:
`Everyone's favorites`, `Must-Haves`, `Try Something New`, The etc. 

- [ ] `Browse all` links need to say (because they don't live immediately close to the title) what you actually are going to browse. Like `Browse all paid plugins`, reinforcing where you go. 

- [ ] `Learn more` isn't enough. Something like `Learn more about plugins in WordPress.com`

- [ ] `Add additional features to your WordPress site. Risk free.` should say `WordPress.com`, right?

And a more higher level of question: why aren’t we promoting more clearly our Jetpack standalone plugins offering (or the ones we can provide explicitly) as a case study of how plugins provide a better, more flexible version of your site? It’s almost a subtle educational/promotional opportunity of the value of plugins. It can also reinforce the association JP/Dotcom.

### PR DESCRIPTION
**Pull request by lsl**
#### Proposed Changes

* Replaces discover page category titles and subheadings
* Updates misc discover page whitespace / margins / placements

#### Testing Instructions

* Load http://calypso.localhost:3000/plugins/
* Should see matching design updates per [this comment](https://github.com/Automattic/wp-calypso/issues/65574#issuecomment-1238156028) and new titles / subtitles per [this comment](https://github.com/Automattic/wp-calypso/issues/65574#issuecomment-1237461756).

Before
Screenshot 2022-09-12 at 18-07-45 Plugins ‹ Business Atomic — WordPress com

After
Screenshot 2022-09-12 at 18-07-39 Plugins ‹ Business Atomic — WordPress com

Fixes https://github.com/Automattic/wp-calypso/issues/65574

### CODE REVIEW

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/7522391

Thank you @lsl for including a screenshot in the description! This is really helpful for our translators.

**Comment by lsl** [comment]
todo: check search result pages per https://github.com/Automattic/wp-calypso/pull/67633#discussion_r968358586

**Comment by lsl** [comment]
@WunderBart / @worldomonation the test changes are title text updates. Feel free to ignore the review request.

File: client/components/section/index.tsx [hunk 1]
```diff
@@ -28,36 +29,49 @@ const SectionContainer = styled.div< SectionContainerProps >`
 		width: 200vw;
 		left: -100vw;
 		z-index: -1;
-		margin-top: -96px;
+		margin-top: -56px;
 	}
-	padding: 96px 0;
+	padding: 56px 0;
```
Comment (lsl) [inline client/components/section/index.tsx:34]: Reduced gap between education sections to match new design figma.

File: client/components/section/index.tsx [hunk 2]
```diff
@@ -28,36 +29,49 @@ const SectionContainer = styled.div< SectionContainerProps >`
...
 		<SectionContainer dark={ dark }>
-			<SectionHeader dark={ dark } className="wp-brand-font">
-				{ header }
-			</SectionHeader>
+			<SectionHeaderContainer>
+				<SectionHeader dark={ dark } className="wp-brand-font">
+					{ header }
+				</SectionHeader>
+				{ subheader && <SectionSubHeader>{ subheader }</SectionSubHeader> }
+			</SectionHeaderContainer>
```
Comment (lsl) [inline client/components/section/index.tsx:74]: New design had a subheading where we didn't have one previously.

File: client/my-sites/plugins/plugins-browser-item/style.scss [hunk 3]
```diff
@@ -7,7 +7,7 @@
 	cursor: pointer;
 	display: block;
 	float: left;
-	margin: 10px 0;
+	margin: 20px 0 0 0;
```
Comment (lsl) [inline client/my-sites/plugins/plugins-browser-item/style.scss:10]: Using margin top to space from the header. Zero'd the bottom margin to differentiate gaps between end of all sections and end of a section on discover. (I think they differed?)

File: client/my-sites/plugins/plugins-browser-item/style.scss [hunk 4]
```diff
@@ -16,20 +16,15 @@
 	}
 
 	.plugin-icon {
-		width: 48px;
-		height: 48px;
+		width: 60px;
+		height: 60px;
```
Comment (lsl) [inline client/my-sites/plugins/plugins-browser-item/style.scss:20]: Merged the override up

File: client/my-sites/plugins/plugins-browser-list/index.jsx [hunk 5]
```diff
@@ -87,22 +87,22 @@ const PluginsBrowserList = ( {
 
 	return (
 		<div className="plugins-browser-list">
-			<div className="plugins-browser-list__header">
-				{ ( title || subtitle ) && (
+			{ ( title || subtitle ) && (
+				<div className="plugins-browser-list__header">
```
Comment (lsl) [inline client/my-sites/plugins/plugins-browser-list/index.jsx:91]: Made header optional as it now has a height. There is an issue where the category pages have two headers. Cleaning up in a follow up https://github.com/Automattic/wp-calypso/pull/67781

File: client/my-sites/plugins/plugins-browser-list/style.scss [hunk 6]
```diff
@@ -53,9 +58,10 @@
 		flex-wrap: wrap;
 		align-items: center;
 		justify-content: space-between;
+		margin-top: auto;
```
Comment (lsl) [inline client/my-sites/plugins/plugins-browser-list/style.scss:66]: Moves/aligns the "Browse All" link actions section to the bottom of the header section whether there is a subheading or not.

File: client/my-sites/plugins/plugins-browser/single-list-view/index.jsx [hunk 7]
```diff
@@ -64,6 +66,7 @@ const SingleListView = ( { category, plugins, isFetching, siteSlug, sites } ) =>
 			plugins={ plugins.slice( 0, SHORT_LIST_LENGTH ) }
 			listName={ category }
 			title={ categoryName }
+			subtitle={ categoryDescription }
```
Comment (lsl) [inline client/my-sites/plugins/plugins-browser/single-list-view/index.jsx:69]: Gives us subheadings for each discover section.

File: client/my-sites/plugins/plugins-category-results-page/header.scss [hunk 8]
```diff
@@ -3,28 +3,29 @@
 
 .plugin-category-results-header {
 	position: relative;
-	top: 36px;
-	padding: 12px 16px 0;
+	padding: 32px 16px 4px 16px;
```
Comment (lsl) [inline client/my-sites/plugins/plugins-category-results-page/header.scss:6]: Matching discover page changes

File: client/my-sites/plugins/plugins-category-results-page/header.tsx [hunk 9]
```diff
@@ -13,7 +13,7 @@ const Header = ( {
 		<div className="plugin-category-results-header">
 			<h1 className="plugin-category-results-header__title">{ title }</h1>
 			{ subtitle && <h2 className="plugin-category-results-header__subtitle">{ subtitle }</h2> }
-			<p className="plugin-category-results-header__count">{ count }</p>
+			<div className="plugin-category-results-header__count">{ count }</div>
```
Comment (lsl) [inline client/my-sites/plugins/plugins-category-results-page/header.tsx:16]: Avoiding `p`'s margin-bottom.

**Comment by DustyReagan** [comment]
Looks good! I noticed there's a "Spotlight" banner that looks like it probably needs a `margin-top: 24px;` [here](https://github.com/Automattic/wp-calypso/blob/c977f31ad3fde516b5f9bc450aaa38a6417ed0c0/client/components/spotlight/index.tsx#L4-L12).

spotlight-top-margin

Also, did we want to update the page title in this PR to "Flex your site's features with plugins" or no?

File: client/my-sites/plugins/plugins-browser/index.jsx [hunk 10]
```diff
@@ -147,7 +149,11 @@ const PluginsBrowser = ( { trackPageViews = true, category, search, hideHeader }
 				isSticky={ isAboveElement }
 				searchTerm={ search }
 				isSearching={ isFetchingPluginsBySearchTerm }
-				title={ translate( 'Plugins you need to get your projects done' ) }
+				title={
+					'en' === locale || hasTranslation( 'Flex your site’s features with plugins' )
+						? __( 'Flex your site’s features with plugins' )
+						: __( 'Plugins you need to get your projects done' )
+				}
```
Comment (lsl) [inline client/my-sites/plugins/plugins-browser/index.jsx:156]: Most of the translations are already back, and I'd rather not block merging over this one that was missed. 

We can fallback on the original one for non en locales in the meantime.

File: test/e2e/specs/plugins/plugins__browse-calypso-user.ts [hunk 11]
```diff
@@ -30,7 +30,7 @@ describe( DataHelper.createSuiteTitle( 'Plugins: Browse' ), function () {
 			await pluginsPage.visit();
 		} );
 
-		it.each( [ 'Top premium plugins', 'Editor’s pick', 'Top free plugins' ] )(
+		it.each( [ 'Must-have premium plugins', 'Our developers’ favorites', 'The free essentials' ] )(
```
Comment (worldomonation) [inline test/e2e/specs/plugins/plugins__browse-calypso-user.ts]: **comment:blocking**: seems to be a few repetitions of strings defined here in subsequent sections.

Have you considered extracting the string definitions into a variable?

**Review by worldomonation — DISMISSED** [review]
Left one blocking comment on behalf of @Automattic/kitkat.

File: test/e2e/specs/plugins/plugins__browse-calypso-user.ts (hunk 11 above)
Comment (lsl) [reply test/e2e/specs/plugins/plugins__browse-calypso-user.ts]: Good idea, see [f139423](https://github.com/Automattic/wp-calypso/pull/67636/commits/f1394239254b7331af59b2b3911fca25cbcdd200)

File: client/components/section/index.tsx (hunk 2 above)
Comment (paulopmt1) [reply client/components/section/index.tsx:74]: Thanks for commenting!

File: client/my-sites/plugins/plugins-browser/index.jsx (hunk 10 above)
Comment (paulopmt1) [reply client/my-sites/plugins/plugins-browser/index.jsx:156]: Sounds good

**Review by paulopmt1 — APPROVED** [review]
Great job @lsl, everything working as expected! Thanks for your comments, helped validating the code.

@vinimotaa is it intentional to have bigger titles for the footer than the rest of the page?

image

**Review by lupus2k — APPROVED** [review]
LGTM!

**Comment by lsl** [comment]
> @vinimotaa is it intentional to have bigger titles for the footer than the rest of the page?

Looks like these should match. Will do in [the follow](https://github.com/Automattic/wp-calypso/pull/67781#issuecomment-1250436102) up after the sprint goal is complete.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/7522391).

**Pull request merged**